File: app/src/main/java/com/rkopylknu/minimaltodo/Reminder/ReminderActivity.java
package com.rkopylknu.minimaltodo.Reminder;

import android.os.Bundle;

import androidx.annotation.NonNull;

import com.rkopylknu.minimaltodo.R;
import com.rkopylknu.minimaltodo.AppDefault.AppDefaultActivity;

public class ReminderActivity extends AppDefaultActivity {

    @Override
    protected void onCreate(final Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected int contentViewLayoutRes() {
        return R.layout.reminder_layout;
    }

    @NonNull
    @Override
    protected ReminderFragment createInitialFragment() {
        return ReminderFragment.newInstance();
    }


}

File: app/src/main/java/com/rkopylknu/minimaltodo/Reminder/ReminderActivity.kt
package com.rkopylknu.minimaltodo.Reminder

import android.os.Bundle
import com.rkopylknu.minimaltodo.R
import com.rkopylknu.minimaltodo.AppDefault.AppDefaultActivity

class ReminderActivity : AppDefaultActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_reminder)

        setSupportActionBar(findViewById(R.id.toolbar))
    }

    companion object {

        const val TO_DO_ITEM_KEY = "to_do_item"
    }
}